ocs_sample_library_preview/Security/OwnerType.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class OwnerType(Enum):
```

Comment:
Why is this distinct from `TrusteeType`? I think in the backend it is the same enumeration.

Reply:
This is another decision that I was on the fence about. I made it a separate class since the type ```Role``` is invalid for Owners. I can remove this and use ```TrusteeType``` if we would rather the user get an error when submitting an invalid type

Reply:
I think since the docs call it `TrusteeType` and that's also what it is in the backend, I would just use that. If it's a value that isn't allowed here, I don't think that's a problem we should solve by branching to a separate enum (unless that's what we did in the backend and docs, at least).

Reply:
I removed the OwnerType enum and substituted the TrusteeType enum